astronomer/providers/amazon/aws/operators/redshift_cluster.py

```diff
@@ -15,6 +16,80 @@
     from airflow.utils.context import Context
 
 
+class RedshiftDeleteClusterOperatorAsync(RedshiftDeleteClusterOperator):
+    """
+    Delete an AWS Redshift Cluster if cluster status is in `available` state.
+
+    :param cluster_identifier: ID of the AWS Redshift Cluster
+    :param aws_conn_id: aws connection to use
+    :param skip_final_cluster_snapshot: determines cluster snapshot creation
+    :param final_cluster_snapshot_identifier: name of final cluster snapshot
+    :param cluster_status_fetch_interval_seconds: interval seconds to wait for fetching cluster status
+    """
+
+    def __init__(
+        self,
+        *,
+        skip_final_cluster_snapshot: bool = True,
+        final_cluster_snapshot_identifier: Optional[str] = None,
+        cluster_status_fetch_interval_seconds: int = 10,
+        aws_conn_id: str = "aws_default",
+        poll_interval: float = 5.0,
```

Comment:
what is the difference between cluster_status_fetch_interval_seconds and poll_interval? I think we usually use poll_interval to fetch the status.

Reply:
using poll_interval now.